Makefile

```diff
@@ -633,6 +633,30 @@ endif
 	git commit -m "Adding $(MLRUN_VERSION) tag contents" --allow-empty; \
 	git push origin $$BRANCH_NAME
 
+.PHONY: test-backward-compatibility-dockerized
+test-backward-compatibility-dockerized: build-test ## Run backward compatibility tests in docker container
+	docker run \
+		-t \
+		--rm \
+        --network='host' \
+        -v /tmp:/tmp \
+        -v /home/runner/work/mlrun/mlrun:/home/runner/work/mlrun/mlrun \
```

Comment:
This path is for the GH action runner, but someone might want to run it on their local env, please verify it's working

Reply:
Are you expecting to be able to run the dockerized-tests in local environment as well and not by the ci workflow?

Reply:
yes